python/gui/requirements.txt

```diff
@@ -10,7 +10,7 @@ pytz==2019.2
 requests==2.22.0
 six==1.12.0
 sqlparse==0.3.0
-urllib3==1.26.5
+urllib3==1.25.11
 websockets
```

Comment:
This was moved to 1.26.5 by #80, which was automatically generated.  So we may see that happen again absent any further adjustments.  

Further, I'm pretty sure these Dependabot-generated PRs are created to help automatically address security vulnerabilities.  Also keep in mind that #78 did something similar a month earlier, but only moved from 1.25.6 to 1.25.8 even though there were versions of 1.26.x available then.  So if it moved to 1.26.5, there was probably good reason, which may or may not apply to our usage.  If we are going to undo this, we need to have a sense of what risk we are potentially taking back on.